src/plugins/interactive_setup/public/progress_indicator.tsx

```diff
@@ -13,7 +13,7 @@ import React, { useEffect, useState } from 'react';
 import useAsyncFn from 'react-use/lib/useAsyncFn';
 import useTimeoutFn from 'react-use/lib/useTimeoutFn';
 
-import type { IHttpFetchError } from '@kbn/core/public';
+import type { IHttpFetchError } from '@kbn/core-http-browser';
```

Comment:
we don't need this change for now, do we? AFAIK, we re-exporting the type in `src/core/plugin/index.ts`. What do you think?

Reply:
We spoke about it with @pgayvallet, and we said that the goal mid-term was to get rid of the facade export in `src/core/public`. This is a step in that direction.

Reply:
Yea, the mid term goal is to get rid of the facade exports. However I would have restrained ourselves from performing this change so soon in the migration to package process for the following reasons (sorry if I did not made myself clear during our previous discussions):
- It brings a **lot** of reviewers, creating delays in the review process 
- It introduce inconsistencies with already migrated domains that are still exported/imported from the entrypoint facade
- There are still risks that some types moves around between packages. Having everything imported from the end packages will cause such refactoring to re-trigger a review from the whole kibana teams

That's why I would have kept rewriting the imports for the cleanup step of the migration.

That being said, now that it's done, I won't ask to revert the changes, it's fine to keep them in the PR.